mne_bids/path.py

```diff
@@ -345,6 +378,27 @@ def copy(self):
         """
         return deepcopy(self)
 
+    def mkdir(self, exist_ok=True, parents=True):
+        """Create the BIDS path directory.
+
+        Parameters
+        ----------
+        exist_ok : bool
+            Whether it is okay that the directory
+            already exists (default is True).
+        parents : bool
+            Whether or not to create parent directories
+             (default is True).
+
+        Returns
+        -------
+        bids_path : pathlib.Path
+            The created directory.
+        """
+        bids_path = Path(self.directory)
+        bids_path.mkdir(exist_ok=exist_ok, parents=parents)
+        return bids_path
```

Comment:
I would return self. You can always get the directory with .fpath.parent

Reply:
i would allow you to chain operations

Reply:
Hmm is that really a common use case, chaining with `mkdir`?

`bids_path.mkdir()....` how would one proceed after that? Typically, one has "massaged" BIDSPath with all the params one needs before calling `mkdir`, no? I guess it could be more convenient to return the created dir:

```python
dir_ = bids_path.mkdir()
dir_.rglob(...)
```

Not saying that I have a very strong opinion here, just brainstorming.

Reply:
> ```python
> bids_path.mkdir().rglob(...)
> ```

Then again, who'd want to glob an empty dir *LOL*

Reply:
This outcome might  depend on how we resolve https://github.com/mne-tools/mne-bids/pull/543#discussion_r482297399

Reply:
I would return self anyhow. You can do .mkdir().directory

now would:

Path(str(bids_path)).parent do the job?

basically I want to keep the public API simple to start with and see
if certain things become recurrent requests.